dataikuapi/dss/projectdeployer.py

```diff
@@ -302,8 +364,19 @@ def get_raw(self):
         """
         return self.settings
 
+    @property
+    def bundle_id(self):
```

Comment:
So we don't want to have a setter in the end?

Reply:
Clément suggested to use a property here as we do for most ids. Users can still set the field with `bundle_id = new_bundle_id`, do you want me to add a comment to make it more explicit?